### PR TITLE
Update to QUnit 2.0 compatible output.

### PR DESCRIPTION
**Pull request by rwjblue**
See http://qunitjs.com/upgrade-guide-2.x/ for more details.

Closes https://github.com/ember-cli/ember-cli/issues/3196.

### CODE REVIEW

File: blueprints/adapter-test/files/tests/unit/__path__/__test__.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ moduleFor('adapter:<%= dasherizedModuleName %>', '<%= classifiedModuleName %>Ada
 });
 
 // Replace this with your real tests.
-test('it exists', function() {
+test('it exists', function(assert) {
```
Comment (jakecraige) [inline blueprints/adapter-test/files/tests/unit/__path__/__test__.js]: Does this and the other places need to be `QUnit.test` ?

**Comment by jakecraige** [comment]
You missed a lot of `test` => `QUnit.test` and a few `module` => `QUnit.module`

File: blueprints/adapter-test/files/tests/unit/__path__/__test__.js (hunk 1 above)
Comment (rwjblue) [inline blueprints/adapter-test/files/tests/unit/__path__/__test__.js]: Test is imported from ember-qunit here.

**Comment by chadhietala** [comment]
Restarting travis but looks good to me.

**Pull request merged**